libclc/clc/lib/generic/math/clc_rsqrt.inc

```diff
@@ -0,0 +1,4 @@
+__attribute__((weak)) _CLC_OVERLOAD _CLC_DEF __CLC_GENTYPE
+__clc_rsqrt(__CLC_GENTYPE val) {
+  return __CLC_FP_LIT(1.0) / __clc_sqrt(val);
```

Comment:
```suggestion
__clc_rsqrt(__CLC_GENTYPE val) {
  #pragma clang fp contract(fast)
  return __CLC_FP_LIT(1.0) / __clc_sqrt(val);
```

Reply:
For this to work as you want, probably needs to directly use the sqrt builtin (or really, we should enable fast contract throughout the entire library and disable it in the handful of locations that are not OK with it)